contracts/DCAHub/DCAHubConfigHandler.sol

```diff
@@ -30,6 +30,7 @@ abstract contract DCAHubConfigHandler is DCAHubParameters, AccessControl, Pausab
   uint16 public platformFeeRatio = 2500; // 25%
 
   mapping(address => bool) internal _allowedTokens;
+  mapping(address => uint120) internal _magnitude;
```

Comment:
Again, why internal? We had a lot of problems in v1 because we couldn't understand the current state and it wasn't queryable. Is there a benefit in making this internal?